nautobot/ipam/tables.py

```diff
@@ -42,28 +42,10 @@
 {% if record.present_in_database %}{% utilization_graph record.get_utilization %}{% else %}&mdash;{% endif %}
 """
 
-PREFIX_LINK = """
```

Comment:
Is this removed because it was unused? My only hesitation is whether we might have any apps that are relying on it that we don't know about.

Reply:
This was not exposed to the apps/plugin, or do plugin developers use constants not exposed to apps?

Reply:
They aren't *supposed* to but that doesn't mean they can't. Just being overly cautious here.